schema/google/showcase/v1beta1/messaging.proto

```diff
@@ -289,7 +289,10 @@ message Blurb {
   option (google.api.resource) = {
     type: "showcase.googleapis.com/Blurb"
     pattern: "rooms/{room_id}/blurbs/{blurb_id}"
+    pattern: "rooms/{room_id}/blurbs/legacy/{legacy_room_id}.{blurb_id}"
```

Comment:
This looks ok to me, although I'm no expert in showcase!

Reply:
👍